Packs/Salesforce/Integrations/SalesforceFusionIAM/SalesforceFusionIAM.py

```diff
@@ -0,0 +1,345 @@
+import demistomock as demisto
+from CommonServerPython import *
+from IAMApiModule import *
+import traceback
+import urllib3
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+ERROR_CODES_TO_SKIP = [
+    404
+]
+
+''' GLOBAL VARIABLES '''
+URI_PREFIX = '/services/data/v51.0/'
+
+'''CLIENT CLASS'''
+
+
+class Client(BaseClient):
+    """ A client class that implements logic to authenticate with the application. """
+
+    def test(self):
+        """ Tests connectivity with the application. """
+
+        uri = URI_PREFIX + 'sobjects/User/testid'
+        self._http_request(method='GET', url_suffix=uri)
+
+    def get_user_by_id(self, user_id: str) -> Optional[IAMUserAppData]:
+        """ Queries the user in the application using REST API by its ID, and returns an IAMUserAppData object
+        that holds the user_id, username, is_active and app_data attributes given in the query response.
+
+        :type user_id: ``str``
+        :param user_id: ID of the user in the application.
+
+        :return: An IAMUserAppData object if user exists, None otherwise.
+        :rtype: ``Optional[IAMUserAppData]``
+        """
+        uri = URI_PREFIX + f'sobjects/FF__Key_Contact__c/{user_id}'
+
+        res = self._http_request(
+            method='GET',
+            url_suffix=uri
+        )
+
+        if res and res.status_code == 200:
+            user_app_data = res.json()
+            username = user_app_data.get('Work_Email__c')
+            is_active = True
+
+            IAMUserAppData(user_id, username, is_active, user_app_data)
+        return None
+
+    def get_user(self, email: str) -> Optional[IAMUserAppData]:
+        """ Queries the user in the application using REST API by its email, and returns an IAMUserAppData object
+        that holds the user_id, username, is_active and app_data attributes given in the query response.
+
+        :type email: ``str``
+        :param email: Email address of the user
+
+        :return: An IAMUserAppData object if user exists, None otherwise.
+        :rtype: ``Optional[IAMUserAppData]``
+        """
+        uri = URI_PREFIX + 'parameterizedSearch/'
+        params = {
+            "q": email,
+            "sobject": "FF__Key_Contact__c",
+            "FF__Key_Contact__c.where": f"Work_Email__c='{email}'",
+            "FF__Key_Contact__c.fields": "Id, FF__First_Name__c, FF__Last_Name__c,Work_Email__c,Name"
+        }
+
+        res = self._http_request(
+            method='GET',
+            url_suffix=uri,
+            params=params
+        )
+
+        if res and res.status_code == 200:
```

Comment:
no need to check the status code - you allow only ok_codes 200 and 201 anyway
```suggestion
        if res:
```